core/message-tags-3.3.md

```diff
@@ -38,7 +38,7 @@ this capability indicate that they are capable of parsing all well-formed tags,
 even if they don't handle them specifically.
 
 Servers advertising this capability indicate that they are capable of handling
```

Comment:
handling -> parsing ?

Mostly because 'handling' sounds like it refers to actually implementing the functionality that every tag out there is supposed to represent, where parsing just means that it can parse it on incoming messages.